sqlite3/lib/src/wasm/database.dart

```diff
@@ -344,6 +344,11 @@ class WasmDatabase extends CommonDatabase {
     );
   }
 
+  @override
+  Future<void> flush() async {
+    await bindings.persistentStorage?.flush();
+  }
```

Comment:
It feels a bit weird to have an API for this on the database to be honest.

I rather change the drift wasm API to also manage the file system and call `flush()` directly instead of doing it through the database here.